### ISSUE TITLE
Pipelines Build Fail to Complete on 8.9.x

### ISSUE THREAD

**Issue by aweingarten** [acquia/blt#1535]
When running a pipelines build on 8.9.x head the build errors out with:  "[error]  TTY mode requires /dev/tty to be read/writable. 
"

```
> validate:phpcs
Iterating over fileset files.php.custom.themes...
.
.
.

.
Setting up local environment...
> setup:build
>> setup:behat
Generating Behat configuration files...
>> setup:composer:install
[error]  TTY mode requires /dev/tty to be read/writable. 
pipelines_kill_background_processes
pstree -p $$ | grep -E -o '([0-9]+)' | grep -v "^$$\$" | sort -k2 -rn
Killing background jobs
```

**Comment by grasmash** [comment]
Addressed in #1534.

**Comment by AllieRays** [comment]
Upgrading to drush 9.7 fixed this for me 

https://github.com/drush-ops/drush/releases/tag/9.7.0
